src/views/Dashboard.vue

```diff
@@ -234,7 +234,7 @@
 
         var objectivesPerDay = Math.ceil(timelineObjectivesComplete / Math.ceil((Date.now() - firstObjectiveComplete) / 86400000))
 
-        const START_REPUTATION = this.$store.get('progress/gameEdition') === 3 ? 0.2 : 0;
+        const START_REPUTATION = this.$store.get('progress/gameEdition') === 4 ? 0.2 : 0;
```

Comment:
```suggestion
        const START_REPUTATION = this.$store.get('progress/gameEdition') >= 3 ? 0.2 : 0;
```
Both 'Prepare for Escape' and 'Edge of Darkness' have the bonus starting reputation